addons/mail/static/src/models/composer/composer.js

```diff
@@ -597,11 +597,12 @@ function factory(dependencies) {
         /**
          * @private
          * @param {string} content html content
-         * @returns {string|undefined} command, if any in the content
+         * @returns {mail.channel_command|undefined} command, if any in the content
          */
         _getCommandFromText(content) {
             if (content.startsWith('/')) {
-                return content.substring(1).split(/\s/)[0];
+                const command_name = content.substring(1).split(/\s/)[0];
```

Comment:
JS variable, should be camel case.

Also I would ideally name it for what it is, which is "user input" or more specifically "first word after /" (if you have a better ideal feel free to suggest).

Because it is not necessarily a command name, which is why this PR is needed in the first place.

Reply:
Hm... indeed naming it `commandName` assume too early that value is a command name, which is not known for sure at this moment.

Could name it `word`. A bit generic, but that's how I would call such string between `/` and whitespace... When considering words are separated with whitespace, which is not true in all locale and without considering punctuation... Maybe you can find something better?